project.clj

```diff
@@ -53,6 +53,10 @@
                               "phantom-advanced" ["phantomjs"
                                                   "src/test/js/runner.js"
                                                   "src/test/html/advanced.html"]}}
+
+  :repl-options {;; The large number of ClojureScript tests is causing long compilation times
```

Comment:
This is a little surprising to me. Maybe something to look into at some point.
What is your "start repl" workflow that you experienced such long build times, just for reference?